### ISSUE TITLE
get pod log request maybe account in the wrong api latency metric report

### ISSUE THREAD

**Issue by WIZARD-CXY** [kubernetes/kubernetes#49998]
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line: 
>
/kind bug
> /kind feature

**What happened**:
use kubectl search log with follow option get api latency (apiserver_request_latencies_summary{quantile="0.99",resource="pods",verb="GET"} )high to give a alert in my cluster.

the apiserver log is below
```
I0802 02:58:31.657224       1 wrap.go:75] GET /api/v1/namespaces/kube-system/pods/ke-logservice-f08fw/log?follow=true: (1m25.080670681s) 200 [[kubectl/v1.6.0 (linux/amd64) kubernetes/fff5156] 10.200.20.65:61803]
```
and we can see it costs 1m25s. 

**What you expected to happen**:

do not make this api latency account in getting pod request api latency

**How to reproduce it (as minimally and precisely as possible)**:
use kubectl to search a pod log with follow option long enough and you will get this metric
apiserver_request_latencies_summary{quantile="0.99",resource="pods",verb="GET"} higher than the ordinary case.
**Anything else we need to know?**:
I am happy to fix it.

**Environment**:
- Kubernetes version (use `kubectl version`): 1.6
- Cloud provider or hardware configuration**: bare-metal
- OS (e.g. from /etc/os-release): ubuntu 16.04
- Kernel (e.g. `uname -a`): 4.4
- Install tools: kubeadm
- Others:

**Comment by mml** [comment]
/assign @WIZARD-CXY

Thanks... looking forward to your PR!

**Comment by WIZARD-CXY** [comment]
@mml fixed with #50123
